data/uk.co.mrbenshef.Boop-GTK.desktop

```
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Name=Boop-GTK
+Comment=Port of IvanMathy's Boop to GTK, a scriptable scratchpad for developers.
+Type=Application
+Exec=boop-gtk
+Terminal=false
+Categories=Utility;GTK;
+Keywords=Rust;Flatpak;GTK;Tools;
```

```suggestion
Keywords=GTK;Tools;
```
not very useful and could mislead the user's research